Automattic/wp-calypso
# Add state to the variation types UI

This PR contains just the state logic, based on code from #12684, that deals with variation types. (So this does not contain the variation generation changes).

The product toggle and variation types are stored in the Redux store so edits can be saved and passed to other future product form components.

There are a few todos here for being able to edit existing products. This PR currently only deals with the add a product flow.

To Test:
* Make sure `make test` passes.
* Make changes to the variation types and make sure they persist.

If needed, state can be cleared with: `localStorage.clear(); indexedDB.deleteDatabase( 'calypso' );`.
